apps/trading/pages/styles.css

```diff
@@ -9,16 +9,30 @@ body,
 }
 
 /* Styles for allotment */
-html {
+html body {
   --focus-border: theme('colors.vega.pink');
   --separator-border: theme('colors.neutral.300');
 }
 
-html.dark {
+html.dark body {
   --focus-border: theme('colors.vega.yellow');
   --separator-border: theme('colors.neutral.600');
 }
 
 .border-default {
   @apply border-neutral-300 dark:border-neutral-600;
 }
+
+html.dark ::-webkit-calendar-picker-indicator {
```

Comment:
Question (blocking): Why style this here? If a calendar picker needs to be inverted in the trading app, surely it needs it elsewhere too.

Reply:
Input styling in the ui-toolkit. You can use the tailwind 'invert' class